File: console_edition_Java/MyChess/Score.java
package MyChess;

/**
 * 各棋形的分数
 */
public class Score {

    /** 连五 */
    final public static int FIVE = 10000000;// 1e7
    /** 活四 */
    final public static int ALIVE_FOUR = 100000;// 1e5
    /** 死四 */
    final public static int BLOCKED_FOUR = 10000;// 1e4
    /** 活三 */
    final public static int ALIVE_THREE = 1000;// 1e3
    /** 死三 */
    final public static int BLOCKED_THREE = 100;// 1e2
    /** 活二 */
    final public static int ALIVE_TWO = 20;// 原100
    /** 死二 */
    final public static int BLOCKED_TWO = 2;// 原10
    /** 活一 */
    //final public static int ALIVE_ONE = 10;// 1e1
    /** 死一 */
    //final public static int BLOCKED_ONE = 1;
    /** 无穷值表示胜利 */
    final public static int INF = (int)1e9;
}

File: src/GameAlgo/Score.js
//各棋形的分数
let Score = {
    /** 连五 */
    FIVE: 1e7,
    /** 活四 */
    ALIVE_FOUR: 1e5,
    /** 死四 */
    BLOCKED_FOUR: 1e4,
    /** 活三 */
    ALIVE_THREE: 1e3,
    /** 死三 */
    BLOCKED_THREE: 1e2,
    /** 活二 */
    ALIVE_TWO: 20,
    /** 死二 */
    BLOCKED_TWO: 2,
    /** 无穷值表示胜利 */
    INF: 1e9
}
export default Score;